tests/monster_test.cpp

```diff
@@ -300,37 +300,45 @@ static void monster_check()
 TEST_CASE( "write_slope_to_speed_map_trig", "[.]" )
 {
     clear_map_and_put_player_underground();
+    bool old_trigdist = trigdist;
```

Comment:
```suggestion
    restore_on_out_of_scope<bool> restore_trigdist( trigdist );
```
There's a helper for this kind of restoration job (from `cata_utility.h`).

Reply:
Thanks! (I can't quite commit the suggestion literally, since it would leave trigdist = old_trigdist there to produce errors...)